lib/src/base-styles/_nx-colors.scss

```diff
@@ -374,6 +396,27 @@
     --nx-color-checkbox-disabled-background: var(--nx-color-disabled);
     --nx-color-checkbox-disabled-inner-border: var(--nx-color-disabled);
     --nx-color-checkbox-disabled-icon: var(--nx-color-component-background);
+
+    // NxToggle
+    --nx-color-toggle-track-disabled: var(--nx-color-disabled);
+    --nx-color-toggle-handle-disabled: var(--nx-color-component-background);
+
+    --nx-color-toggle-off-border: var(--nx-swatch-indigo-70);
+    --nx-color-toggle-off-border-hover: var(--nx-swatch-indigo-60);
+  
+    --nx-color-toggle-off-track: var(--nx-color-component-background);
+    --nx-color-toggle-off-track-hover: var(--nx-color-component-background);
```

Comment:
In Sketch, the background of track is set to `nx-indigo-15` on hover, should we use `--nx-color-site-background` instead?

Reply:
> In Sketch, the background of track is set to `nx-indigo-15` on hover, should we use `--nx-color-site-background` instead?

Ooo thanks for catching that! Fixed here: b840cd5b345ed805ce908e55bfce7e070616cc58